docs/sdkx_android/users.mdx

```diff
@@ -102,33 +106,42 @@ You should call Helpshift SDK's Login API whenever the user successfully logs in
 
 <Admonition type="info" title="Note">
 
-In order to create a logged-in user in Helpshift, the use of either `userId` or `userEmail` is required.
+- In order to create a logged-in user in Helpshift, the use of either `userId` or `userEmail` is required.
+- If you are setting `fullPrivacy` to true, you shouldn't use email as the only identifier in the login API for that user. This will result in the creation of an anonymous user.
 
 </Admonition>
 
-```java
-    Map<String, Object> userData = new HashMap<>();
-    userData.put("userId", "12234454");
-    userData.put("userEmail", "captain@example.com");
-    userData.put("userName", "Captain Planet");
+Example:
 
-    Helpshift.login(userData);
+```java
+Map<String, Object> userData = new HashMap<>();
+userData.put("userId", "12234454");
+userData.put("userEmail", "captain@example.com");
+userData.put("userName", "Captain Planet");
+Helpshift.login(userData);
 ```
 
 ### Logging out users {#logging-out-users}
 
 #### Overview
 
 Once a user logs out from your app, you should call the Helpshift SDK logout API to ensure no one else can view this user's conversations.
-_Note:_ The logout API is expected to be used in conjunction with the login API.
 
-#### How to use
+<Admonition type="info" title="Note">
+
+- The logout API is expected to be used in conjunction with the login API.
```

Comment:
We can also add a note here for having unique user id across all the users of the customer and that the user id should not be duplicated for 2 different users